schematic/utils/df_utils.py

```diff
@@ -95,7 +93,13 @@ def convert_ints(x: str) -> Union[np.int64, bool]:
     Returns:
         x converted to type int if possible, otherwise False
     """
-    return np.int64(x) if str.isdigit(x) else False
+    str_is_digit = False
+
+    cell_is_str = isinstance(x, str)
+    if cell_is_str:
+        str_is_digit = str.isdigit(x)
+
+    return np.int64(x) if (cell_is_str and str_is_digit) else False
```

Comment:
This functions logic was really confusion to me. I think it would be much clearer like:

```
if isinstance(x, str) and str.isdigit(x):
  return np.int64(x)
return False